components/input/Input.cs

```diff
@@ -286,6 +405,7 @@ private void ToggleClearBtn()
             {
                 builder.OpenComponent<Icon>(31);
                 builder.AddAttribute(32, "Type", "close-circle");
+                builder.AddAttribute(32, "Theme", "fill");
```

Comment:
Are you sure you would using the same sequenceId?

Reply:
Good catch. Obviously not 😄 . After so many changes I stopped seeing obvious mistakes...

Reply:
😂